### ISSUE TITLE
Using custom databases for annotation

### ISSUE THREAD

**Issue by adityabandla** [geronimp/enrichM#93]
Is there an option for users to provide their own databases for the annotation step. I would like to use more recent versions of PFAM and dbCAN for my dataset

**Comment by adityabandla** [comment]
@geronimp @wwood  Is this repo still being actively maintained?

**Comment by geronimp** [comment]
@adityabandla Yes, you can simply replace the pfam.hmm and cazy.hmm databases in the enrichM database file with the most recent versions. Repo is still maintained yes - have recently been on post phd vacation and I am still catching up on the backlog

**Comment by adityabandla** [comment]
@geronimp Thanks! Congrats on the PhD, great to hear

**Comment by adityabandla** [comment]
Is there a way to recreate all the mapping files that are included in a database release, for example pfam_to_clan or pfam_to_description?
